path/fount.ps1

```
@@ -1615,9 +1615,158 @@ public class ExplorerRefresher {
+					$current = Get-FountTestLidAc
+					if ($null -ne $current -and $current -ne 0) { Set-FountTestLidAc 0 }
+				}
+			}
+			if ($PID -notin $state.holders) { $state.holders = @($state.holders) + $PID }
+			$script:FountTestLidHolder = $true
+		}
+	}
+}
+function Disable-FountTestKeepAwake {
+	if ($script:FountTestKeepAwakeActive) {
+		try { [void][FountKeepAwake]::SetThreadExecutionState([Convert]::ToUInt32('80000000', 16)) } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)